src/developTest/java/mServer/crawler/AddToFilmlistTest.java

```diff
@@ -0,0 +1,123 @@
+package mServer.crawler;
+
+import java.io.IOException;
+import mSearch.daten.DatenFilm;
+import mSearch.daten.ListeFilme;
+import okhttp3.mockwebserver.*;
+
+import org.junit.*;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+public class AddToFilmlistTest {
+    private static final String FILM_NAME_ONLINE = "onlinefilm.mp4";
+    private static final String FILM_NAME_ONLINE2 = "onlinefilm2.mp4";
+    
+    private static MockWebServer mockServer;
+    private static String baseUrl;
+    
+    private ListeFilme list;
+    private ListeFilme listToAdd;
+    
+    @BeforeClass
+    public static void setUpClass() throws IOException {
+        mockServer = new MockWebServer();
+        Dispatcher dispatcher = new Dispatcher() {
+
+            @Override
+            public MockResponse dispatch(RecordedRequest request) throws InterruptedException {
+
+                switch(request.getPath()) {
+                    case "/" + FILM_NAME_ONLINE:
+                    case "/" + FILM_NAME_ONLINE2:
+                        return new MockResponse().setResponseCode(200);
+                }
+                return new MockResponse().setResponseCode(404);
+            }
+        };
+        mockServer.setDispatcher(dispatcher); 
+        mockServer.start();
+        baseUrl = mockServer.url("").toString();
+    }
+    
+    @AfterClass
+    public static void teardownClass() throws IOException {
+        mockServer.shutdown();
+    }
+
+    @Before
+    public void setUp() {
+        listToAdd = new ListeFilme();
+        list = new ListeFilme();
+        list.add(new DatenFilm("BR", "Topic 1", "urL", "Title 1", "http://film1.mp4", "", "", "", 12, ""));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Title 1" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Topic 1" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)